command/ibft/ibft_candidates.go

```diff
@@ -9,50 +9,51 @@ import (
 	empty "google.golang.org/protobuf/types/known/emptypb"
 )
 
-// IbftCandidates is the command to query the snapshot
+// IbftCandidates is the command to get a current list of IBFT candidates
+// and their corresponding votes
 type IbftCandidates struct {
 	helper.Meta
 }
 
 // GetHelperText returns a simple description of the command
-func (p *IbftCandidates) GetHelperText() string {
+func (c *IbftCandidates) GetHelperText() string {
```

Comment:
Another example, this function is very clear in its intentions, no need for the comment above since it brings no value to understanding what the method does.

Reply:
Completely agree 👍 

All of our command interface methods are documented in this way, just to keep them standardized.

We can discuss removing these useless comments for all commands on some future PR

Reply:
agreed.